audio/pulseaudio/Portfile

```diff
@@ -75,6 +75,7 @@ platform darwin {
 # and posts on the mailing list.
 patchfiles          patch-man-Makefile.am.diff \
                     patch-src_modules_macosx_module_coreaudio_device.c-respect-PA_NAME_MAX.diff \
+                    patch-src_modules_macosx_module_coreaudio_device.c-import-util-h.diff \
```

Comment:
@rseichter Thanks for your efforts so far, and sorry for my contentiousness (I'm a busybody, not a MacPorts member).

One more remark: this particular group of patchfiles is intended for an older issue, so the new patch is better off somewhere separate (with optional brief explanation; link to trac ticket should be sufficient):

```
# https://trac.macports.org/ticket/61107
patchfiles-append   patch-src_modules_macosx_module_coreaudio_device.c-import-util-h.diff
```

Reply:
agreed!

Reply:
> sorry for my contentiousness (I'm a busybody, not a MacPorts member).

Like Yoda said: "The Nitpicking is strong with this one." 🙂 Well, I have made that change as well. Now how about somebody merge this beast?